Document that the responsibility for owning drills is now in teams (#4849)
alphagov/govuk-developer-docs: source/manual/tech-lead-responsibilities.html.md

Essentially a tech lead’s role is to be the face that represents all the standards and principles by which developers need to work with.

## Being prepared for incidents and high profile events

> The sorts of things that used to be drilled on Technical 2nd Line are [now the responsibility of the product teams](https://docs.google.com/document/d/1I4o4hLjpjUMthTOXEbmecBqpHvms2HV3EXKCHSCAK3U/edit). In other words, processes like "drilling deploying the emergency banner" are now the responsibility of the publishing teams.

Reviewer: ```suggestion
The sorts of things that used to be drilled on Technical 2nd Line are [now the responsibility of product teams](https://docs.google.com/document/d/1I4o4hLjpjUMthTOXEbmecBqpHvms2HV3EXKCHSCAK3U/edit). For example, a process like "drilling deploying the emergency banner" is now the responsibility of the publishing teams.
```
